ProcessLib/ThermoMechanicalPhaseField/CreateThermoMechanicalPhaseFieldProcess.cpp

```diff
@@ -0,0 +1,313 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CreateThermoMechanicalPhaseFieldProcess.h"
+
+#include <cassert>
```

Comment:
Please check, which includes are needed (in all files), and remove the unnecessary ones.

Reply:
I see you have found some other includes, but missed this `cassert` include.....